src/TodoList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import TodoListItem from './TodoListItem';
+
+function TodoList({
+  todos,
+  onClick,
+}) {
+  return (
+    <>
+      {todos.length
+        ? (
+          <ol>
+            {todos.map(({ key, value }) => (
+              <TodoListItem
+                key={key}
+                todo={value}
+                onClick={() => onClick(key)}
+              />
+            ))}
+          </ol>
+        )
+        : <p>할 일이 없어요!</p>}
```

Comment:
```suggestion
      {todos.length === 0 ? <p>할 일이 없어요!</p> : (
          <ol>
            {todos.map(({ key, value }) => (
              <TodoListItem
                key={key}
                todo={value}
                onClick={() => onClick(key)}
              />
            ))}
          </ol>
        )}
```
list가 없을 경우 처리하는 생각은 좋습니다! 다만 순서를 바꿔서 조금 더 명확하게 처리하는건 어떨까요?

Reply:
순서를 바꾸게 되면 어떤 부분에서 더 명확해지는지 궁금합니다!

Reply:
조건문의 경우, 비교하는 값이 boolean 타입이면 조금 더 이해하기 쉬운 부분이 있어서요 ㅎㅎ 

타입스크립트를 사용할때도, 삼항연산자는 해당이 안되지만,
```typescript
const todos: [] = [];

...
return(
  {todos.length && <p>hi</p>}
);
```
위와 같이 사용하면, 화면에 0이 노출 되버립니다. `todos.length`의 type이 `number`라서죠.
제가 말씀드린 부분은 습관을 잡아두면 좋겠다는 생각에서 였습니다 ☺️

Reply:
아 설명 감사합니다! 이해했어요!
아무래도 조건문은 boolean 타입으로 하는게 맞겠네요!